pkg/contexts/ocm/utils/resource.go

```diff
@@ -16,3 +20,51 @@ func GetResourceData(acc ocm.ResourceAccess) ([]byte, error) {
 	defer m.Close()
 	return m.Get()
 }
+
```

Comment:
Hmmm. This seems to be an unrelated change? And these don't appear to be used. What is the purpose of these?

Reply:
I changed the contract to make it easier, but then I have to do an explicit  close, here.,